src/20241007_Multi_GHOStewardV2Upgrade/GHOStewardV2Upgrade.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "GHO Steward v2 Upgrade"
+author: "@karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-gho-steward-v2-upgrade/19116"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xc5e7df1536ef9fc71a7d2e2f6fee6e4e20e37a50b4e0f1646616d066b8697da5"
+---
+
+## Simple Summary
+
+This publication proposes upgrading the GHO Steward Role to incorporate additional functionality to accomodate the current and future growth of GHO.
+
+## Motivation
+
+In response to the expanding GHO ecosystem, GhoSteward v2 incorporates several different stewards to avoid the need to redeploy the entire steward contract whenever an upgrade or change is proposed.
+
+- GhoBucketSteward
+- GhoAaveSteward
+- GhoCcipSteward
+- GhoGsmSteward
+
+Any future change to the GHO Steward functionality will require only the corresponding steward to be updated. This reduces the complexity and streamlines future amendments to the GHO Steward role.
+
+In addition, some new features have been added to allow for controlling parameters related to CCIP.
+
+## Specification
+
+The following contracts must be granted these roles by the DAO:
+
+- GhoAaveSteward
+  1. RiskAdmin in Aave V3 Ethereum Pool
+- GhoBucketSteward (both on Ethereum and Arbitrum)
+  1. GhoTokenBucketManagerRole on GhoToken
+- GhoCcipSteward
+  1. RateLimitAdmin and BridgeLimitAdmin roles on GhoTokenPool (just rateLimitAdmin on Arbitrum)
+- GhoGsmSteward
+  1. Configurator in every GSM asset that the DAO wants the risk council to manage
+
+To facilitate the CCIP Steward, a new CCIP token pool implementation will be implemented on Arbitrum to allow setting of rateLimitAdmin.
+
+List of new addresses:
+
+| Contract           | Arbitrum                                   | Ethereum                                   |
+| ------------------ | ------------------------------------------ | ------------------------------------------ |
+| New CCIP Token     | 0xb78eDA33EB5493d56f14a81023CE69438a562A2c |                                            |
```

Comment:
More accurate to call it `New CCIP Token Pool`?

Reply:
`New CCIP Token Pool Implementation`